system/cfme-setup.sh

```diff
@@ -31,21 +33,29 @@ cat <<'EOF' > /etc/httpd/conf.d/ssl.conf
 # upgraded.
 EOF
 
+SOURCE_IPTABLES="$MIQ_ROOT/system/TEMPLATE/etc/sysconfig/iptables"
+SOURCE_CTRL_ALT_DEL="$MIQ_ROOT/system/TEMPLATE/etc/init/control-alt-delete.override"
+
 # Incorporate the iptable updates
-if [[ -f /var/www/miq/system/TEMPLATE/etc/sysconfig/iptables ]]; then
+if [[ -f $SOURCE_IPTABLES ]]; then
   /sbin/iptables-save > /etc/sysconfig/iptables.SYSTEM
-  /sbin/iptables-restore /var/www/miq/system/TEMPLATE/etc/sysconfig/iptables
+  /sbin/iptables-restore $SOURCE_IPTABLES
   /sbin/service iptables save
 fi
 
-/usr/sbin/semanage fcontext -a -t httpd_log_t "/var/www/miq/vmdb/log(/.*)?"
-/usr/sbin/semanage fcontext -a -t cert_t "/var/www/miq/vmdb/certs(/.*)?"
-/usr/sbin/semanage fcontext -a -t logrotate_exec_t /var/www/miq/system/logrotate_free_space_check.sh
+# Copy the ctrl-alt-del override if it doesn't exist
+if [[ ! -f /etc/init/control-alt-delete.override ]] && [[ -f $SOURCE_CTRL_ALT_DEL ]] ; then
```

Comment:
rpm does this for us. only caveat, we need to remove the symbolic linking work 

why not:
```
git mv system/LINK/etc/init/control-alt-delete.override system/COPY/etc/init/control-alt-delete.override
```
tell rpm to remove our symbolic link stuff before:
```
%preinstall
[ -L /etc/init/control-alt-delete.override -a "`readlink /etc/init/control-alt-delete.override`" = "/var/www/miq/system/LINK/etc/init/control-alt-delete.override" ] && rm /etc/init/control-alt-delete.override
```
tell rpm not to blow away customer's control-alt-delete.override
```
%config(noreplace) /etc/init/control-alt-delete.override
```